core/src/main/resources/templates/core/management_user_rating.html

```diff
@@ -116,5 +173,28 @@ <h2>Manage <strong>User rating statistics</strong></h2>
     </div>
 </div>
 <div id="footer" th:insert="core/footer"></div>
+
+<!-- Include Date Range Picker -->
+<script type="text/javascript"
+        src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.4.1/js/bootstrap-datepicker.min.js"></script>
+<link rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.4.1/css/bootstrap-datepicker3.css"/>
+
+<script>
+    $('.input-daterange input').each(function () {
+        $(this).datepicker({
+            format: 'yyyy-mm-dd',
+            todayHighlight: true,
+            autoclose: true,
+            orientation: 'top'
+        });
+    });
+
+    function exportFiltered() {
+        document.getElementById("searchForm").action = "/management/rating/exportFiltered";
```

Comment:
Imagine `.submit()` will fail, the action won't be returned to previous value.
It's better to create two functions (bind them to events) each submitting to its own url.